Tools/gen_manifest

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import subprocess
+import re
+from shutil import copyfile
+runtime_libs = []
+def init_runtime_libs () :
+  runtime_libs.append('libc')
+  runtime_libs.append('libdl')
+  runtime_libs.append('libm')
+  runtime_libs.append('libpthread')
+  runtime_libs.append('libutil')
+  runtime_libs.append('libnss_dns')
+  runtime_libs.append('libresolv')
+  runtime_libs.append('librt')
+  
+def filter_default_libs (lib_name) :
+  res = True
+  for lib in runtime_libs :
+    if lib_name == lib :
+      res = False
+  return res
+
+def parse_libs (bin_path) :
+  print (bin_path)
+  ldd_out = subprocess.check_output(['ldd', bin_path])
+  lib_list = []
+  for line in ldd_out.splitlines():
+    match = re.match(r'\t(.*) => (.*) \(0x', line)
+    if match and match.group(1) and match.group(2):
+      full_lib_name = match.group(1)
+      name_match = re.match(r'([\w\d]*)(\.*)(.*)', full_lib_name)
+      if name_match:
+        lib_name = name_match.group(1)
+        lib_path = match.group(2)
+        if filter_default_libs (lib_name) : 
+          lib_list.append((name_match.group(1), match.group(2)))
+  return lib_list
+
+def make_exec(path) :
+  mode = os.stat(path).st_mode
+  mode |= (mode & 0o444) >> 2    # copy R bits to X
+  os.chmod(path, mode)
+
+def gen_manifest(app_name, bin_name, g_path) :
+  m_path = g_path + '/graphene/LibOS/shim/test/apps/' + app_name + "/" + app_name +"." + "manifest"
+  print (m_path)
+  mf = open(m_path, "w")
+  make_exec(m_path)
+  mf.write("#!" + g_path + "/graphene/Runtime/pal_loader \n")
+  mf.write("loader.preload = file:../../../../../Runtime/libsysdb.so \n")
+  
+  # Get Path of Binary
+  bin_path = subprocess.check_output(['which', bin_name]).strip()
+  mf.write('loader.exec = file:' + bin_path + '\n')
+  mf.write('loader.execname = ' + bin_name + '\n')
+  mf.write('loader.env.LD_LIBRARY_PATH = /graphene:/graphene/resolv:/host:/usr/local/lib:/usr/lib:/usr/lib/x86_64-linux-gnu \n')
```

Comment:
IMO it would be nice to have the Graphene's mount point inside the container configurable. What do you think?

Reply:
Agree. Generally, we need to provide an option allowing users to use their own customized manifest files. Will implement this after this PR is merged.